### ISSUE TITLE
The convergence process is very slow.

### ISSUE THREAD

**Issue by keithkang1986** [ldpape/iOLS_delta#2]
I have nearly 300,000 observations. I use iOLS_delta_HDFE. It seems that the convergence process is very slow. I have not yet obtained an estimate after more than 2000 iterations.

**Comment by mycroft1018** [comment]
> I have nearly 300,000 observations. I use iOLS_delta_HDFE. It seems that the convergence process is very slow. I have not yet obtained an estimate after more than 2000 iterations.

Indeed, I encountered a similar situation. I spent more than 10 hours on this（nearly 7million obs）, yet I got no results.

**Comment by mycroft1018** [comment]
> I have nearly 300,000 observations. I use iOLS_delta_HDFE. It seems that the convergence process is very slow. I have not yet obtained an estimate after more than 2000 iterations.

Indeed, I encountered a similar situation. I spent more than 10 hours on this（nearly 7million obs）, yet I got no results.
